Integrations/integration-ProtectWise.yml

```diff
@@ -1,25 +1,18 @@
 description: Cloud based Security Network DVR
-detaileddescription: |-
-  Integration with ProtectWise cloud product.
-  Please note you can use either token (acquired by api call : https://api.protectwise.com/momfodhxhz/#access-token-token) or user email/password for authentication.
+detaileddescription: Integration with ProtectWise cloud product.
```

Comment:
Did you deliberately remove all the detailed description? if adding back make sure to use proper markdown syntax for URLs